api/pkg/apis/v1alpha1/managers/activations/activations-cleanup-manager.go

```diff
@@ -34,16 +32,19 @@ func (s *ActivationsCleanupManager) Init(context *contexts.VendorContext, config
 		return err
 	}
 
-	// Set activation cleanup interval after they are done. If not set, use default 60 minutes.
-	if val, ok := config.Properties["RetentionInMinutes"]; ok {
-		s.RetentionInMinutes, err = strconv.Atoi(val)
+	// Set activation cleanup interval after they are done. If not set, use default 180 days.
+	if val, ok := config.Properties["RetentionDuration"]; ok {
+		s.RetentionDuration, err = time.ParseDuration(val)
```

Comment:
Shall we check if this is the positive value?

Reply:
I have checked negative duration at line 40 and an exception is thrown.

Reply:
Sorry, I missed this line.